core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AppSparkMetricsAnalyzer.scala

```diff
@@ -473,7 +471,7 @@ class AppSparkMetricsAnalyzer(app: AppBase) extends AppAnalysisBase(app) {
         tasksInStage.map(_.sr_totalBytesRead).sum,
         tasksInStage.map(_.sw_bytesWritten).sum,
         tasksInStage.map(_.sw_recordsWritten).sum,
-        shuffleWriteTimeSum
+        TimeUnit.NANOSECONDS.toMillis(shuffleWriteTimeSum) // nanoseconds
```

Comment:
nit:
`// nanoseconds` -> It took me a minute until I realized that this is a comment and not a division :)
Can we move that to be in a separate line above the conversion

Reply:
thanks, removed this.